pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires      = ["setuptools>=61.0.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "siphotonics"
+name = "sipkit"
 dynamic = ["version"]
 description = "A toolkit to design of silicon photonic components"
```

Comment:
"A toolkit to rapidly lookup parameters for the design of silicon photonic components with automatic differentiation capability"